lib/main.dart

```diff
@@ -29,7 +29,8 @@ void main() async {
 
   debugPrint("Starting main app[debugPrint]: ${Config.instance.application}");
 
-  const environment = String.fromEnvironment('ENVIRONMENT');
+  Config.instance.environment.length == 0 ? String.fromEnvironment('ENVIRONMENT') : Config.instance.environment;
```

Comment:
Why not use directly the Config.instance.environment which already runs the String.fromEnvironment ?

Reply:
You're right we can.
Config.instance.environment already check the value is correct. My bad.

Reply:
```suggestion
  const environment = Config.instance.environment;
```